content/v1.14/cli/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+weight: 400
+title: Crossplane CLI
+description: "Documentation for the Crossplane command-line interface"
+---
+
+The Crossplane CLI helps simplify some development and administration aspects of
+Crossplane.
+
+The Crossplane CLI includes:
+* tools to build, install, update and push Crossplane Packages
+* standalone Composition Function testing and rendering without the need to access a Kubernetes cluster running Crossplane
+* visual dependency mapping across Compositions, Composite Resources and Managed Resources
+
+## Installing the CLI
+
+The Crossplane CLI is a single standalone binary with no external dependencies.
+
+{{<hint "note" >}}
+Install the Crossplane CLI on a user's computer. 
+
+The Crossplane CLI is independent of Kubernetes and doesn't require a Crossplane
+pod.
+{{< /hint >}} 
+
+To download the latest version for your CPU architecture with the Crossplane
+install script.
+
+```shell
+curl -sL "https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh" | sh
+```
+
+[The script](https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh)
+detects your CPU architecture and downloads the latest stable release.
+
+Manually download a binary from the Crossplane releases repository at 
+https://releases.crossplane.io/stable/current/bin
+
+{{<hint "important" >}}
+<!-- vale write-good.Passive = NO -->
+The CLI is named `crank` in the release repository. Download this file. 
+<!-- vale write-good.Passive = YES -->
```

Comment:
Could we get away without this, and have everyone use the script? It would help avoid explaining the crank thing.

Reply:
I would but since some people don't like `curl | bash` they would look at the repo and it's not obvious that `crank` is the binary, not `crossplane`. This exact scenario tripped me up.

Reply:
Uploading all binaries to the gh releases too as many other tools do could allow us not to have to point people at the s3 bucket.